_posts/2022-03-07-fine-dining-for-indices.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title:  "Fine Dining for Indices"
+authors:
+- nateboot
+
+date: 2022-03-07
+categories:
+ - intro
+
+excerpt: "There are a lot of reasons you might want to fine-tune indices on an OpenSearch cluster. Every workflow has different requirements, and the default behavior of OpenSearch might not suit your use case. Before ingesting with reckless abandon to fill an index with data, enjoy some insight on tuning an index. "
+---
+
+## Is Your Cluster Eating Healthy? 
+
+There are a lot of reasons you might want to fine-tune indices on an OpenSearch cluster.  Every workflow has different requirements, and the default behavior of OpenSearch might not suit your use case.  Before ingesting with reckless abandon to fill an index with data, enjoy some insight on tuning an index.  A small bit of foresight can help fine tune a cluster for speed or space efficiency to suit a particular workflow. 
+
+## Are You Going To Eat All of That? 
+
+By default, OpenSearch indexes all the fields in each document it ingests. Enabling or disabling this on a per field basis allows for a customized index with only specific fields in it. This does not nullify the original document - a field that isn’t indexed can still be retrieved and parsed from the `_source` field, but the data is not part of the index, nor can it be aggregated by visualizations.  Consider this dev tool snippet -
```

Comment:
'OpenSearch indexes all the fields in each document it ingests' and 'a field that isn’t indexed' -> the term "index" (and variants) are really over loaded in OpenSearch (it's a noun, verb, etc.). Maybe rephrase for clarity.

Reply:
Let me know if the latest push alleviates your concerns on clarity.